src/pages/create/SetFromPage.tsx

```
@@ -60,6 +62,27 @@ export default function SetFromPage() {
     });
   };
 
+  const normalizeFromName = (name: string) =>
+  name.trim().replace(/\s+/g, ' ').toLowerCase();
```

ESLint

[error] Delete `⏎·`

(prettier/prettier)